tests/create.go

```diff
@@ -57,3 +60,28 @@ func testCreateRestartAlways(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err, out)
 	builderName = strings.TrimSpace(out)
 }
+
+func testCreateRemoteContainer(t *testing.T, sb integration.Sandbox) {
+	if sb.Name() != "docker" {
+		t.Skip("skipping test for non-docker workers")
+	}
+
+	var builderName string
+	t.Cleanup(func() {
+		if builderName == "" {
+			return
+		}
+		out, err := rmCmd(sb, withArgs(builderName))
+		require.NoError(t, err, out)
+	})
+
+	out, err := createCmd(sb, withArgs("--driver", "docker-container"))
+	require.NoError(t, err, out)
+	builderName = strings.TrimSpace(out)
+
+	out, err = inspectCmd(sb, withArgs("--bootstrap", builderName))
+	require.NoError(t, err, out)
+
+	out, err = createCmd(sb, withArgs("--driver", "remote", fmt.Sprintf("docker-container://%s_%s0", driver.BuilderNamePrefix, builderName), "--bootstrap"))
+	require.NoError(t, err, out)
```

Comment:
For a good measure I think better to do `--name <randomname>` and then extra check with  `inspect <randomname>`. Maybe also check that the remote builder is in the running state.

Reply:
Updated to return builder name from output and inspect after.

Reply:
Check that the second builder is in running state as well.